fidelius-service/src/main/java/org/finra/fidelius/services/aws/DynamoDBService.java

```diff
@@ -88,29 +86,6 @@ public <T>List<T> queryDynamoDB(DynamoDBQueryExpression queryRequest, Class<T> c
             logger.info(String.format("Query completed in %.3f seconds", (System.currentTimeMillis() - startTime) / 1000.0));
         }
 
-        return queryResults;
-    }
-
-    // Creates a new DynamoDBMapper object
-    public DynamoDBMapper createMapper(String account, String region, String tableName) {
```

Comment:
We don't need mapping?

Reply:
DynamoDBMapper is no longer supported in SDK v2. Needed to do some workarounds to keep the logic as is.